proposals/0079-allow_commission_decrease_at_any_time.md

```diff
@@ -0,0 +1,83 @@
+---
+simd: '0079'
+title: Allow Commission Decrease at Any Time
+authors:
+  - Bryan Ischo (bryan@ischo.com)
+category: Standard
+type: Core
+status: Draft
+created: 2023-10-26
+feature: TBD
+---
+
+## Summary
+
+The commission_updates_only_allowed_in_first_half_of_epoch feature disallows
+commission decrease in the second half of epoch. Given that the purpose of this
+feature was to prevent 'rug pulls' which are accomplished by increasing
+commission at the end of epoch and then decreasing commission at the beginning
+of next epoch, disallowing decreases during the second half of epoch is
+unnecessary.
+
+A feature gate must be added to support this SIMD as all validators' vote
+programs must treat commission change instructions the same or else consensus
+will diverge.
+
+## Motivation
+
+Some validator operators may need to decrease commission in order to satisfy
+their own operational criteria.  Speaking personally, when I have an operator
+error that results in reduced stake account rewards for the epoch, I reduce
+commission to 0% to ensure that stake accounts are not disadvantaged by my
+actions. Not being allowed to do this in the second half of an epoch is a real
+problem for me.
+
+Allowing commission changes during the second half of epochs also enables
+"reverse rug pulls" where a validator has commission set to 100% during the
+majority of an epoch but reduces it to 0% across the epoch boundary.  It's not
+clear why some validator operators choose to do this, but it does not seem
+harmful, and there does appear to be motivation for some validators to do this
+as it's been done numerous times in the past.
```

Comment:
Well now this makes me worry about a potential security issue for making the change! I don't think there is a problem either, but I'll definitely think it over.

The only one I can come up with: when an entity provides validator-operation-as-a-service for whales, I believe they typically run 100% commission validators and pay out the inflation rewards through some separate channel. If they can "reverse rug", then they can artificially decrease the amount that goes back to their client.  Since they already have a separate channel for setting up this agreement though, I don't think it's a problem.

Reply:
Agreed, there's no concern about that.